src/interfaces/IAuth.ts

```diff
@@ -9,3 +9,10 @@ export interface IAuthResponse {
   email: string
   token: string
 }
+
+export type IAuthContextState = IAuthResponse
```

Comment:
@jgcmarins About typings, is it ok to create an intermediary type that is the same as another interface just to `rename`?

Reply:
talking about types I like to place all of them close to the component that will use it. I prefer to avoid interfaces folder, for example

```
type ComponentProps = {};
const Component = ({ ... }: ComponentProps) => {};
```

Reply:
export directly from there if necessary